valik kod/lab2.cpp

```diff
@@ -0,0 +1,161 @@
+#include <iostream>
+#include <initializer_list>
+#include <functional>
+#include <memory>
+#include <stdexcept>
+
```

Comment:
Naming Conventions: Renamed methods and variables to follow a more descriptive and conventional style, such as get_size, clear_list, and new_node.

Reply:
ok